content/posts/2021-12-10---Podstawowe-operacje.md

```diff
@@ -0,0 +1,321 @@
+
+```java
+if (warunek) {
+  // blok kodu wykonywany w przypadku spełnienia warunku
+}
+```
+```java
+if (warunek) {
+  // blok kodu wykonywany w przypadku spełnienia warunku
+} else {
+  // blok kodu wykonywany w przypadku nie spełnienia warunku
+}
+```
+```java
+if (warunek) {
+  // blok kodu wykonywany w przypadku spełnienia warunku
+} else if (warunek_alternatywny) {
+  // blok kodu wykonywany w przypadku spełnienia warunku alternatywnego
+} else {
+  // blok kodu wykonywany w przypadku nie spełnienia żadnego warunku
+}
+```
+Przykład użycia instrukjcji `if-else` prezentuje się następująco
+```java
+int a = 10;
+int b = 25
+
+if (a > 20) {
+  System.out.println("A jest większe od 20");
+} else {
+  System.out.println("A nie jest większe od 20");
+}
+
+if (b > 20) {
+  System.out.println("B jest większe od 20");
+} else {
+  System.out.println("B nie jest większe od 20");
+}
+```
+Wyjście programu:
+```
+A nie jest większe od 20
+B jest większe od 20
+```
+
+#### Instrukcja `switch-case`
+`Switch-case` jest bardziej rozbudowaną formą wcześniej zaprezentowanej instrukcji `if-else`.
+Została ona stworzona do obsługiwania wielu alternatywnych warunków dotyczących tej samej zmiennej.  
+Deklaruje się ją za pomocą słowa kluczowego `switch` oraz podania wyrażenia, którego dotyczyć będzie ta instrukcja. Przypadki, czyli alternatywne wyniki wyrażenia deklarujemy za pomocą zwrotu `case`. Opcjonalnie na samym końcu możemy umieścić blok kodu, który wykona się w przypadku niedopasowania żadnego z wyżej zadeklarowanych przypadków. Blok ten musimy poprzedzić zwrotem `default`.  
+Deklaracja pełnej instrukcji `switch-case` wygląda następująco:
+
+```java
+switch(wyrazenie) {
+  case x:
+    // kod wykonywany gdy zajdzie przypadek x
+    break;
+  case y:
+    // kod wykonywany gdy zajdzie przypadek y
+    break;
+  default:
+    // przypadek domyślny wykonywany gdy przypadki x oraz y nie zostaną dopasowane do wyrażenia
+}
+```
+Zauważ, że po każdym przypadku na samym końcu umieszczamy instrukcję `break`. Gdybyśmy jej tam nie umieścili to po wykonaniu przypadku `x`, program mógłby wykonać również przypadek `y` lub przypadek domyślny.  
+Przykład dzialania instrukcji `switch-case`:
+```java
+int dzienTygodnia = 4;
+switch (dzienTygodnia) {
+  case 1:
+    System.out.println("Poniedziałek");
+    break;
+  case 2:
+    System.out.println("Wtorek");
+    break;
+  case 3:
+    System.out.println("Środa");
+    break;
+  case 4:
+    System.out.println("Czwartek");
+    break;
+  case 5:
+    System.out.println("Piątek");
+    break;
+  case 6:
+    System.out.println("Sobota");
+    break;
+  case 7:
+    System.out.println("Niedziela");
+    break;
+  default:
+    System.out.println("Błąd! Dzień tygodnia musi być z przedziału 1-7!!!");
+}
+```
+Wyjście programu:
+```
+Czwartek
+```
+
```

Comment:
Jakeiś zakończenie by się przydało. Możesz coś wspomnieć o tym, że to koniec, ale powinniście wyczekiwać kolejnych części czy coś w tym stylu